lib/assets/javascripts/turbograft/response.coffee

```diff
@@ -17,7 +17,19 @@ class TurboGraft.Response
     else
       throw new Error("Error encountered for XHR Response: #{this}")
 
+  redirectedTo: () ->
+    @xhr.getResponseHeader('X-XHR-Redirected-To')
+
+  redirectedToNewUrl: () ->
+    Boolean(
+      @redirectedTo() &&
+      @redirectedTo() != TurboGraft.Response.currentLocation()
```

Comment:
Seems unecessary to have a method to get this when you could just access the current location (unless this is for a test...)